games/godot/Portfile

```diff
@@ -48,67 +76,90 @@ build.env-append    BUILD_NAME=macports_build
 build.target        platform=osx arch=${build_arch} target=release_debug
 
 destroot {
+    if {$subport eq "${name}-3.2"} {
+        set appname "Godot 3.2"
+        set cliname $subport
+    } else {
+        set appname "Godot"
+        set cliname ${name}
+    }
     copy ${worksrcpath}/misc/dist/osx_tools.app \
-         ${destroot}${applications_dir}/Godot.app
-    xinstall -d ${destroot}${applications_dir}/Godot.app/Contents/MacOS
+         ${destroot}${applications_dir}/${appname}.app
+    xinstall -d ${destroot}${applications_dir}/${appname}.app/Contents/MacOS
     move ${worksrcpath}/bin/godot.osx.opt.tools.${build_arch} \
-         ${destroot}${applications_dir}/Godot.app/Contents/MacOS/Godot
-    ln -s ${applications_dir}/Godot.app/Contents/MacOS/Godot \
-       ${destroot}${prefix}/bin/godot
+         ${destroot}${applications_dir}/${appname}.app/Contents/MacOS/Godot
+    ln -s ${applications_dir}/${appname}.app/Contents/MacOS/Godot \
+       ${destroot}${prefix}/bin/$cliname
 }
 
-livecheck.type      regex
-livecheck.url       https://godotengine.org/download/osx
-livecheck.regex     Godot <em>(\\d+\\.\\d+(?:\\.\\d+)*)</em>
+if {$subport eq ${name}} {
+    livecheck.regex releases/tag/(\\d+\\.\\d+(?:\\.\\d+)*)-stable
+} else {
+    livecheck.type  none
+}
 
 ### The following code was adapted from the portfiles
 ### of ports such as openvdb and openimageio
 
 # Get major version from version string
 proc get_major {version} { return [lindex [split ${version} .] 0] }
 
-set clang_versions {5.0 6.0 7.0 8.0 9.0}
+set clang_versions {5.0 6.0 7.0 8.0 9.0 10 11}
 set clang_variants {}
-foreach v ${clang_versions} { lappend clang_variants clang[get_major ${v}] }
+foreach v ${clang_versions} {
+    lappend clang_variants clang[get_major ${v}]
+}
 
 # this default version should stay synchronized with
 # the default macports_clang option in the SConstruct
 # file that is part of the Godot source code
 set sconstruct_default_mp_clang 5
 
 # error: invalid value 'gnu++14' in '-std=gnu++14'
-# On older OSX versions use (latest) clang variant by default
-if { ${os.major} <= 13 } {
+# On older macOS versions use (latest) clang variant by default
+if {$subport eq ${name} && ${os.major} <= 15 || \
```

Comment:
check for `os.platform` here and possibly elsewhere in the Portfile as well

Reply:
Completed requested change